Mainframe3270/py3270.py

```diff
@@ -235,7 +220,7 @@ def __init__(self, extra_args):
             "-script",
         ]
         if extra_args:
```

Comment:
I am still thinking about a way to remove these redundant `if extra_args:` statements.